packages/babel-plugin-remove-graphql-queries/src/index.ts

```diff
@@ -1,10 +1,60 @@
 /*  eslint-disable new-cap */
-const graphql = require(`gatsby/graphql`)
-const { murmurhash } = require(`./murmur`)
-const nodePath = require(`path`)
+import { graphql } from "gatsby"
```

Comment:
I don't think this is a correct change. This imports "graphql" tag from gatsby, while the original line `require("gatsby/graphql")` actually imported graphql tools, including `parse`, etc.

```suggestion
import graphql from "gatsby/graphql"
```

Reply:
To clarify, `import graphql from "gatsby/graphql"` imports everything from this file: https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/graphql.js

But `import { graphql } from "gatsby"` imports this function: https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/cache-dir/gatsby-browser-entry.js#L92

Reply:
That's a great catch